build.gradle

```diff
@@ -8,7 +8,7 @@ sourceCompatibility = 1.6
 
 group = 'com.box'
 archivesBaseName = 'box-java-sdk'
-version = '1.1.0'
+version = '1.2.0-snapshot'
```

Comment:
I think snapshot needs to be all caps for maven to recognize it - `1.2.0-SNAPSHOT`